Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/TorsionForceField.inl

```diff
@@ -118,6 +120,28 @@ void TorsionForceField<DataTypes>::addKToMatrix(linearalgebra::BaseMatrix* matri
 	}
 }
 
+template <typename DataTypes>
+void TorsionForceField<DataTypes>::buildStiffnessMatrix(core::behavior::StiffnessMatrix* matrix)
+{
+    auto dfdx = matrix->getForceDerivativeIn(this->mstate)
+                       .withRespectToPositionsIn(this->mstate);
```

Comment:
remove the return line?

Reply:
This line break is actually in all force field that I already implemented. I added this line break to mimic the math notation. `dfdx` actually represents

$$\frac{df}{dx}$$

The first line (`matrix->getForceDerivativeIn(this->mstate)`) corresponds to $df$, and the second line (`.withRespectToPositionsIn(this->mstate)`) is $dx$

Reply:
ok 👍